configs/components/cfpropertylist.rb

```diff
@@ -1,7 +1,7 @@
 component "cfpropertylist" do |pkg, settings, platform|
   pkg.version "2.2.7"
   pkg.md5sum "be2fc619084422cc28cb68f6084bea91"
-  pkg.url "http://buildsources.delivery.puppetlabs.net/CFPropertyList-2.2.7.tar.gz"
+  pkg.url "https://rubygems.org/downloads/CFPropertyList-#{pkg.get_version}.gem"
```

Comment:
That might actually be me just messing up the upstream url. I mean, I'm glad the redirect thing is now fixed, but I should probably just point to the tarball upstream, if that's possible.